### PR TITLE
livekit-cli 1.0.2 (new formula)

### PR DESCRIPTION
**Pull request by davidzhao**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/livekit-cli.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  homepage "https://livekit.io"
+  url "https://github.com/livekit/livekit-cli/archive/refs/tags/v1.0.1.tar.gz"
+  sha256 "87196347bdbad99daf404f7fa3f6e58e431ad829d1db6765b7f6d7d81ede7aa0"
+  license "Apache-2.0"
+  head "https://github.com/livekit/livekit-cli.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(output: bin/"livekit-cli"), "./cmd/livekit-cli"
```
Comment (carlocab) [inline Formula/livekit-cli.rb:12]:
```suggestion
    system "go", "build", *std_go_args, "./cmd/livekit-cli"
```

Comment (davidzhao) [reply Formula/livekit-cli.rb]: done

File: Formula/livekit-cli.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/livekit-cli"
+  end
+
+  test do
+    output = shell_output("#{bin}/livekit-cli create-token --list --api-key key --api-secret secret")
+    assert output.start_with?("valid for (mins):  5")
+  end
```
Comment (chenrui333) [inline Formula/livekit-cli.rb:18]: add version check, currently it has [some issue](https://github.com/livekit/livekit-cli/pull/36)
```suggestion

    assert_match "livekit-cli version #{version}", shell_output("#{bin}/livekit-cli --version")
  end
```

Comment (davidzhao) [reply Formula/livekit-cli.rb:18]: actually that wouldn't play well with brew cache.. I'll release 1.0.2 instead.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@davidzhao, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**